evm/deploy-gateway-v6.2.x.js

```diff
@@ -368,6 +368,8 @@ async function deploy(config, chain, options) {
     contractConfig.tokenDeployer = tokenDeployer.address;
     contractConfig.deployer = wallet.address;
     contractConfig.deploymentMethod = options.deployMethod;
+    contractConfig.connectionType = options.amplifier ? 'amplifier' : 'consensus';
```

Comment:
let's remove amplifier support from this script (that was for the outdated gateway)